docs/design-language.md

```diff
@@ -330,3 +330,70 @@ There are four types of notifications available to use:
+
+---
+
+## Typography
+
+Good typography plays a large role in creating not only an aesthetically pleasing app but a usable one. It is essential that style guidelines and semantics are followed as they ensure consistency across our suite of applications. For example, by keeping headings and body text at predefined sizes makes it easier for developers to implement. There is no guessing required as to which size an H1 should be. Users working with our applications will instantly know they are using a VertiGIS app simply but the way it looks (and behaves).
+
+### Type Scale
+
+<div class="typography buttons-box">
+    <h1>h1. Heading (2rem/32px, weight: 300)</h1>
+    <h2>h2. Heading (1.75rem/28px, weight: 300)</h2>
+    <h3>h3. Heading (1.625rem/26px, weight: 400)</h3>
+    <h4>h4. Heading (1.375rem/22px, weight: 400)</h4>
+    <h5>h5. Heading (1.250rem/20px, weight: 600)</h5>
+    <h6>h6. Heading (1rem/16px, weight: 600)</h6>
+    <h6 class="subtitle1">subtitle1 (1rem/16px, weight: 400, letter-spacing: 0.00938rem). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur</h6>
+    <h6 class="subtitle2">subtitle2 (0.875rem/14px, weight: 600, letter-spacing: 0.01071rem). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur</h6>
+    <p class="body1">body1 (0.875rem/14px). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos laborum fugiat deleniti? Eum quasi quidem quibusdam.</p>
+    <p class="body2">body2 (0.75rem/12px). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos laborum fugiat deleniti? Eum quasi quidem quibusdam.</p>
+    <p class="caption">Caption text (0.875rem/14px)</p>
+</div><br />
+
+### Font Stack
+
+Our current font stack is <em>"Segoe UI, Helvetica Neue, Roboto, Helvetica, Arial, sans-serif"</em>. **<a href="https://learn.microsoft.com/en-us/typography/font-list/segoe-ui" target="_blank">Segoe UI</a>** is installed by default on Windows. MacOS/iOS will take advantage of Helvitca Neue and Roboto is Android's default font. **Font sizes rely on the browser default font-size of 16px. This value should not be changed.**
+
+### Headings
+
+Headings are the largest text on the screen and are used for titles. They help organize content and demonstrate importance using a hierarchy that ranges from 1-6. For more information regarding headings please see **<a href="https://www.w3.org/WAI/tutorials/page-structure/headings/" target="_blank">W3C</a>** and **<a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements" target="_blank">MDN Web Docs</a>**.
+
+#### Considerations when using Headings
+
+-   Do not alter the weight of headings using CSS or HTML tags.
```

Comment:
Should we also tell people to not alter the size along with the weight?

Reply:
Yeah!

Reply:
Updated.

Reply:
A new thought. We should advise people to not chose a heading based on the size that they want (because that could change in future MUI updates) and that they should chose a heading level based on it's the heirarchy of the content.

Reply:
Updated.